app/src/components/adminConsole/currentCup/CurrentCupHeader.tsx

```diff
@@ -5,7 +5,7 @@ const CurrentCupHeader: React.FunctionComponent = (props: {
     isOpen: Boolean;
 }) => {
     return (
-        <div>
+        <div id="currentCupHeader">
```

Comment:
What's this id for?

Reply:
it was from attempting to remove popups, has now been removed